packages/jellyfish-api-core/__tests__/category/poolpair/updatePoolPair.test.ts

```diff
@@ -0,0 +1,54 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers/dist/index'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { poolpair, BigNumber } from '@defichain/jellyfish-api-core'
+import { Testing } from '@defichain/jellyfish-testing'
+import { RpcApiError } from '../../../src'
+
+describe('poolpair update', () => {
+  const container = new MasterNodeRegTestContainer()
+  const testing = Testing.create(container)
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should update a pool', async () => {
```

Comment:
Properly you should do a `setup` of `beforeEach()`, creating a new container for each scenario. 

Separately assert each, field that is being updated.

```ts
expect(object).toStrictEqual({
  pool: '',
  ABC: expect.any(String),
  txid: expect.stringMatching(/^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/)
})
```

Prefer if to use an object matcher assert, we started to deprecate the use of individual assertions.